CHANGES.txt

```diff
@@ -1,3 +1,6 @@
+Release 0.7.41 - 2017-08-04
+  - Change TDClient#appendTableSchema to not send column alias for append-schema API [#97]
```

Comment:
Perhaps you meant `[#97](https://github.com/treasure-data/td-client-java/pull/97)`?